python/cuml/cuml/naive_bayes/naive_bayes.py

```diff
@@ -1129,73 +1076,26 @@ class MultinomialNB(_BaseDiscreteNB):
 
     Examples
     --------
-
     Load the 20 newsgroups dataset from Scikit-learn and train a
     Naive Bayes classifier.
 
-    .. code-block:: python
-
-        >>> import cupy as cp
-        >>> import cupyx
-        >>> from sklearn.datasets import fetch_20newsgroups
-        >>> from sklearn.feature_extraction.text import CountVectorizer
-        >>> from cuml.naive_bayes import MultinomialNB
-
-        >>> # Load corpus
-        >>> twenty_train = fetch_20newsgroups(subset='train', shuffle=True,
-        ...                                   random_state=42)
-
-        >>> # Turn documents into term frequency vectors
-
-        >>> count_vect = CountVectorizer()
-        >>> features = count_vect.fit_transform(twenty_train.data)
-
-        >>> # Put feature vectors and labels on the GPU
-
-        >>> X = cupyx.scipy.sparse.csr_matrix(features.tocsr(),
-        ...                                   dtype=cp.float32)
-        >>> y = cp.asarray(twenty_train.target, dtype=cp.int32)
-
-        >>> # Train model
-
-        >>> model = MultinomialNB()
-        >>> model.fit(X, y)
-        MultinomialNB()
-
-        >>> # Compute accuracy on training set
-
-        >>> model.score(X, y)
-        0.9245...
-
+    >>> from sklearn.datasets import fetch_20newsgroups
+    >>> from sklearn.feature_extraction.text import CountVectorizer
+    >>> from cuml.naive_bayes import MultinomialNB
+    >>> data = fetch_20newsgroups(subset='train', shuffle=True, random_state=42)
+    >>> X = CountVectorizer().fit_transform(data.data)
+    >>> model = MultinomialNB().fit(X, data.target)
+    >>> model.score(X, y)  # doctest: +SKIP
+    0.9245...
     """
 
-    def __init__(
-        self,
-        *,
-        alpha=1.0,
-        fit_prior=True,
-        class_prior=None,
-        output_type=None,
-        handle=None,
-        verbose=False,
-    ):
-        super(MultinomialNB, self).__init__(
-            alpha=alpha,
-            fit_prior=fit_prior,
-            class_prior=class_prior,
-            handle=handle,
-            output_type=output_type,
-            verbose=verbose,
-        )
```

Comment:
Why is the constructor removed here?

Reply:
It's redundant, the parent class constructor is identical.